streaming_jupyter_integrations/magics.py

```diff
@@ -371,6 +378,17 @@ def __enrich_cell(self, cell: str) -> str:
     def __retract_user_as_something_is_executing_in_background() -> None:
         print("Please wait for the previously submitted task to finish or cancel it.")
 
+    @staticmethod
+    def __read_global_config() -> dict:
+        if "FLINK_HOME" in os.environ:
+            with open(os.path.join(os.environ["FLINK_HOME"], "conf", "flink-conf.yaml"), 'r') as stream:
+                try:
+                    parsed_yaml = yaml.safe_load(stream)
+                    return parsed_yaml
+                except yaml.YAMLError as exc:
+                    print(exc)
+        return {}
```

Comment:
Here I would put a warning `FLINK_HOME environment variable is not set, could not find a 'flink-conf.yaml'` or something like that